config/locales/en.yml

```diff
@@ -1,4 +1,51 @@
 en:
+  coronavirus_pages:
+    actions:
+      discard_changes: Discard changes
+      view_on_govuk: View on GOV.UK
+      preview: Preview
+      publish: Publish
+    announcements:
+      confirm: Are you sure?
+      reorder: Reorder
+      add: Add announcement
+    draft:
+      heading: 3. Update draft
+      button_text: Update draft
+    github:
+      update_heading: 1. Update GitHub
+      review_heading: 2. Get changes reviewed and merged
+        update_sub_heading_html: "Update the <a href=\"%{github_url}\">coronavirus content on Github</a>"
```

Comment:
@JonathanHallam I think all of the tests are failing because this line is indented too far.